.github/workflows/trigger-nightly.yml

```diff
@@ -0,0 +1,27 @@
+name: Trigger nightly builds
+
+on:
+  schedule:
+    # every night at 4:30AM
+    - cron: 30 4 * * *
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - .github/workflows/trigger-nightly.yml
+permissions:
+  contents: write
+
+jobs:
+  tag_nightly:
+    if: ${{ github.repository == 'pytorch/vision' }}
+    name: Tag nightly from main
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: main
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Tag nightly from main
+        run: |
+          git tag -f nightly-build
```

Comment:
The comment in `build-cmake.yml` states

```
# Nightly builds look like: nightly-2022-06-03
```

but here we are just pushing `nightly-build`. Please correct one or the other.

Reply:
@pmeier yes I opted for floating tag: nightly-build